webots_ros2_tutorials/protos/RobotSenseCamera.proto

```diff
@@ -0,0 +1,228 @@
+#VRML_SIM R2020b utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# tags: static
+# More info here: https://www.youtube.com/c/soft-illusion
+# Robotsense
+
+PROTO RobotSenseCamera [
+    field SFVec3f    translation  0 0.0198 0
+    field SFRotation rotation     0 1 0 2.3495
+    field SFString   controller   "<extern>"
+  ]
+  {
+    Robot {
+      translation IS translation
+      rotation IS rotation
+      controller IS controller
+      children [
+        Camera {
+          translation 0 0.03 0.1
+        rotation 0 -1 0 3.14158
+      children [
```

Comment:
There is wrong indentation starting from here.

Reply:
We were using different photos because we had to scale to suit our world. 
Name of texture was changed to "soft_illusion" was changed.
Intendation updated in RobotSenseCamera.proto